directory.td.json

```diff
@@ -181,6 +184,64 @@
                 }
             ]
         },
+        "retrieveTDs": {
+            "description": "Paginate and retrieve Thing Descriptions",
+            "uriVariables": {
+                "offset": {
+                    "description": "The number of TDs to skip before the collection",
+                    "type": "number",
+                    "default": 0
+                },
+                "limit": {
+                    "description": "The number of TDs in the collection",
+                    "type": "number",
+                    "default": 1 
```

Comment:
@sebastiankb @mmccool Is there a way to set default values in TD? 
The [URI Template](https://tools.ietf.org/html/rfc6570) does not support it after [draft v5](https://tools.ietf.org/html/draft-gregorio-uritemplate-05#section-2.5) and I couldn't find anything in the TD's DataSchema either.

Reply:
Best to create an issue about this in the TD repo, otherwise it may not get addressed.

Reply:
Thanks. I'll reply on the new issue: https://github.com/w3c/wot-thing-description/issues/1093